src/View/CrudView.php

```diff
@@ -6,6 +6,58 @@
 
 class CrudView extends View
 {
+    public $layout = 'CrudView.default';
+
+    public function initialize()
+    {
+        parent::initialize();
+        $this->_setupBootstrapUI();
+        $this->_setupViewblocks();
+    }
+
+    protected function _setupBootstrapUI()
+    {
+        $this->loadHelper('Html', ['className' => 'BootstrapUI.Html']);
+        $this->loadHelper('Form', ['className' => 'BootstrapUI.Form']);
+        $this->loadHelper('Flash', ['className' => 'BootstrapUI.Flash']);
+        $this->loadHelper('Paginator', ['className' => 'BootstrapUI.Paginator']);
+    }
+
+    protected function _setupViewblocks()
+    {
+        $viewblocks = $this->get('viewblocks', []);
+        foreach ($viewblocks as $viewblock => $set) {
+            $output = '';
+            foreach ($set as $key => $type) {
+                if ($type == 'element') {
+                    $output .= $this->element($key);
+                } elseif ($type == 'Html::css') {
+                    $output .= $this->Html->css($key);
+                } elseif ($type == 'Html::script') {
+                    $output .= $this->Html->script($key);
+                } else {
+                    $output .= $key;
+                }
+            }
+            $this->Blocks->set($viewblock, $output);
+        }
+    }
+
+    /**
+     * Finds an element filename, returns false on failure.
+     *
+     * @param string $name The name of the element to find.
+     * @return mixed Either a string to the element filename or false when one can't be found.
+     */
+    protected function _getElementFileName($name)
```

Comment:
I believe overriding methods which return templates path can be avoided if the extra lookup paths are just added to config var `App.path.templates`.